guide/14-deep-learning/how_deeplabv3_works.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<blockquote>&nbsp;guide "How U-net Works" where an intuition of Semantic Segmentation explained.</blockquote><p>looks like an incomplete statement</p><p> </p><blockquote>To understand contents of this guide, we assume that you have some basic understanding of the convolutional neural networks (CNN) concepts. You can refresh your CNN knowledge by going through this short paper “A guide to convolution arithmetic for deep learning”.</blockquote><p>Should go under Note section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>